aiida/transport/plugins/ssh.py

```diff
@@ -483,7 +485,7 @@ def open(self):
         # Open a SSHClient
         connection_arguments = self._connect_args
         proxystring = connection_arguments.pop('proxy_command', None)
-        if proxystring is not None:
+        if proxystring:
```

Comment:
Maybe change to `if proxystring != ''`. However, we should really set it to `None` if it is not specified.